package.json

```diff
@@ -10,10 +10,11 @@
     "url": "https://github.com/mxenabled/react-native-widget-sdk.git"
   },
   "scripts": {
-    "prepack": "npm run build:dist",
-    "documentation:copy": "cp node_modules/@mxenabled/widget-post-message-definitions/docs/generated.md docs/widget_callback_props.md",
-    "build": "tsc",
-    "build:dist": "[ -d dist ] && rm -r dist; npm run build -- --declaration --noEmit false --outDir dist",
+    "prepack": "npm run build",
+    "clean": "[ -d dist ] && rm -r dist || true",
+    "compile": "tsc --declaration --noEmit false --outDir dist",
+    "build": "npm run clean && npm run compile && npm run documentation",
+    "documentation": "cp node_modules/@mxenabled/widget-post-message-definitions/docs/react-native-sdk-generated.md docs/widget_callback_props.md",
```

Comment:
`npm run documentation` used to be `npm run documentation:copy` -- we can reintroduce the namespacing if we have more "documentation" commands in the future.